{"uuid": "57f94e4f-1cac-40f2-b5ff-aebaedffe20f", "counter": 6, "stdout": "\u001b[1;35m[WARNING]: Unhandled error in Python interpreter discovery for host\u001b[0m\r\n\u001b[1;35memydinfi110: Failed to connect to the host via ssh: ssh: connect to host\u001b[0m\r\n\u001b[1;35m206.189.197.112 port 22: Connection refused\u001b[0m", "start_line": 5, "end_line": 8, "runner_ident": "0ed2eaa0-2a36-45ce-944e-05c9f6fa8a9c", "event": "warning", "pid": 792015, "created": "2020-12-04T14:49:49.057281", "parent_uuid": "00d8619c-d37a-346a-c3b0-000000000094", "event_data": {"playbook": "main.yml", "playbook_uuid": "0bf2d6d9-b0ee-4a34-8522-592d4aa8a6e5", "play": "VPS Hardening", "play_uuid": "00d8619c-d37a-346a-c3b0-000000000006", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "00d8619c-d37a-346a-c3b0-000000000094", "task_action": "gather_facts", "task_args": "", "task_path": "/home/craig/git/tempor/tempor/playbooks/main.yml:2", "warning": true, "uuid": "57f94e4f-1cac-40f2-b5ff-aebaedffe20f"}}